src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx

```diff
@@ -190,7 +190,7 @@ describe.each([
     ;(supportsETH as unknown as jest.Mock<unknown>).mockReturnValue(true)
 
     const sendClose = jest.fn()
-    ;(ensLookup as unknown as jest.Mock<unknown>).mockImplementation(async () => ({
+    ;(ensLookup as unknown as jest.Mock<unknown>).mockImplementation(() => ({
```

Comment:
`ensLookup` is of type `((domain: string) => Promise<ResolveVanityAddressReturn>) & MemoizedFunction`, I reckon this should return a
```suggestion
    ;(ensLookup as unknown as jest.Mock<unknown>).mockImplementation(() => Promise.resolve({
```

Reply:
Checks out!